driver.go

```diff
@@ -59,33 +81,37 @@ func (driver linodeVolumeDriver) List() (*volume.ListResponse, error) {
 // Create implementation
 func (driver linodeVolumeDriver) Create(req *volume.CreateRequest) error {
 	log.Info("Create(%s)", req.Name)
-
-	vol, err := driver.linodeAPI.CreateVolume(req.Name, req.Options)
+	// @TODO "Size: req.Options..."" how to specify? default to linode default for now (20G).. This is easier.
```

Comment:
req.Options is a map that captures any arbitrary options passed in the command line as `-o <key>=<value>`.  It comes from the command line at the time of Volume creation

```
docker volume create -o size=50 -d linode-driver my-test-volume-50
```